components/drops/BasicDropCard.vue

```diff
@@ -134,11 +134,11 @@ const props = withDefaults(
 const { placeholder } = useTheme()
 const isUnlimited = computed(() => props.dropMax > Number.MAX_SAFE_INTEGER)
 
-const chainPropList = chainPropListOf(props.dropPrefix)
+const chainPropList = chainPropListOf(props.dropPrefix ?? 'ahp')
```

Comment:
doesn't the prop already have this default set ? , if you still need it here make it a constant and also use it inside deineProps 

```suggestion
const chainPropList = chainPropListOf(props.dropPrefix ?? DEFAULT_DROP_PREFIX)
```

Reply:
oh correct, I removed `null` in the props instead `dropPrefix?: Prefix | null`